geti_sdk/prediction_visualization/visualizer.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (C) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+"""The package provides the Visualizer class for models predictions visualization."""
+
+
+from typing import Optional
+
+import cv2
+import numpy as np
+
+from geti_sdk.data_models.annotation_scene import AnnotationScene
+from geti_sdk.prediction_visualization.shape_drawer import ShapeDrawer
+
+
+class Visualizer:
+    """
+    Visualize the predicted output by drawing the annotations on the input image.
+
+    :example:
+        >>> predictions = inference_model.predict(frame)
+        >>> annotation = prediction_converter.convert_to_annotation(predictions)
+        >>> output = visualizer.draw(frame, annotation.shape, annotation.get_labels())
+        >>> visualizer.show(output)
```

Comment:
Nice catch! Do you think this example would work?

```suggestion
        >>> prediction = deployment.infer(image)
        >>> visualizer = Visualizer()
        >>> output_image = visualizer.draw(image, prediction )
        >>> Display(output_image)
```

Reply:
Yep, looks good!